centaur/src/main/resources/standardTestCases/jes_labels.test

```diff
@@ -12,8 +12,8 @@ metadata {
   workflowName: I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_
   status: Succeeded
   "calls.I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_.my_task_aliased.backendLabels.cromwell-workflow-id": "cromwell-<<UUID>>"
-  "calls.I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_.my_task_aliased.backendLabels.wdl-task-name": "my-task"
-  "calls.I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_.my_task_aliased.backendLabels.wdl-call-alias": "my-task-aliased"
+  "calls.I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_.my_task_aliased.backendLabels.wdl-task-name": "my_task"
+  "calls.I_hope_nobody_names_workflows_like_this_as_it_seems_very_unnecessary_.my_task_aliased.backendLabels.wdl-call-alias": "my_task_aliased"
```

Comment:
It looks like the purpose of this test was to ensure that strings that were valid WDL identifiers but invalid Google labels were properly aliased, but with the expanded label patterns that is no longer being tested.

Reply:
If I'm reading the WDL spec right, it looks like an acceptable identifier which is an unacceptable label would contain capital letters or end with an underscore. If that's right, I can just change that test case

Reply:
sounds good 🙂